### PR TITLE
feat:Added color picker for noteCard and edit functionality

### PR DESCRIPTION
**Pull request by Hardiegogo**
Added :
-  hexColorPicker from react-colorful
-  edit note functionality

image

### CODE REVIEW

**Review by nerdyspook — APPROVED** [review]
Code structure looks good, and the UI is really good!

File: src/components/notecard-component/NoteCard.jsx [hunk 1]
```diff
@@ -1,26 +1,37 @@
 import axios from "axios";
-import React from "react";
+import React,{useState} from "react";
 import { useNotes } from "../../context/useNotes";
 import "./NoteCard.css";
 import { addOrRemoveFromArchive } from "../../utils/notes-util/addOrRemoveFromArchive";
 import { addToTrash } from "../../utils/trash-util/addToTrash";
+import { HexColorPicker } from "react-colorful";
+import { changeColor } from "../../utils/notes-util/changeColor";
```
Comment (nerdyspook) [inline src/components/notecard-component/NoteCard.jsx:8]: You can create a js file to export all the util js files and import from there wherever you need when you refactor the code.

Comment (Hardiegogo) [reply src/components/notecard-component/NoteCard.jsx:8]: Got it.

**Review by stutisk — APPROVED** [review]
code looks clean 
liked the way you have organised folder s
got to learn something new

**Pull request merged**